content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -429,6 +430,59 @@ memory      0     20Gi
 pods        0     10
 ```
 
+### Cross Namespace Pod Affinity Quota
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Operators can use `CrossNamespacePodAffinity` quota scope to limit which namespaces are allowed to have pods 
+with affinity terms that cross namespaces (i.e., allowed to set `namespaces` or `namespaceSelector` fields in pod affinity terms)
+
+Preventing users from using cross-namespace affinity terms might be necessary since a pod with anti-affinity constraint can block pods from all other namespaces 
```

Comment:
```suggestion
Preventing users from using cross-namespace affinity terms might be desired since a pod with anti-affinity constraints can block pods from all other namespaces 
```